pytorch/buildspec.yml

```diff
@@ -23,6 +28,16 @@
       example_mnist_file:
         source: docker/build_artifacts/mnist.py
         target: mnist.py
+    inference_context: &INFERENCE_CONTEXT
```

Comment:
Build inference images for testing

Reply:
So this change will revert in final change

Reply:
yes it will be